appeals/functions/lpaq-import/local.settings.json

```diff
@@ -0,0 +1,8 @@
+{
+	"IsEncrypted": false,
+	"Values": {
+		"FUNCTIONS_WORKER_RUNTIME": "node",
+		"AzureWebJobsStorage": "UseDevelopmentStorage=true",
+		"API_HOST": "http://localhost:3000"
```

Comment:
Should API_HOST contain the protocol as the POST request has it hardcoded to https?